src/workerd/io/features.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <workerd/io/compatibility-date.capnp.h>
+#include <workerd/jsg/jsg.h>
+
+namespace workerd {
+
+struct FeatureFlags {
+  FeatureFlags() = delete;
+
+  static CompatibilityFlags::Reader get(jsg::Lock&);
```

Comment:
It's a little weird that there's a class called `FeatureFlags` and a class called `CompatibilityFlags`, where `FeatureFlags` isn't actually a thing you instantiate, it just exists to have this static method, and `FeatureFlags::get()` doesn't return an instance of `FeatureFlags`, it returns `CompatibilityFlags::Reader`...

Of course, the problem is we can't put new methods in `CompatibilityFlags` because it's a generated type. But maybe a freestanding `getCompatibilityFlags(js)` function might be easier for readers to understand?